test/extended/builds/dockerfile.go

```diff
@@ -88,5 +92,26 @@ USER 1001
 			_, err = oc.Client().ImageStreamTags(oc.Namespace()).Get("centos", "7")
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("should be able to start a build from Dockerfile with FROM reference to scratch", func() {
+			g.By("calling oc new-build with Dockerfile that uses scratch")
+			err := oc.Run("new-build").Args("-D", "-").InputString(testDockerfile3).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("starting a test build")
+			bc, err := oc.Client().BuildConfigs(oc.Namespace()).Get("scratch")
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(*bc.Spec.Source.Dockerfile).NotTo(o.BeNil())
```

Comment:
I'm confused, are the tests not checking the content of the Dockerfile on the next line?
`o.Expect(*bc.Spec.Source.Dockerfile).To(o.Equal(testDockerfile3))`